File: euler/prob5.c
#include <stdio.h>
#include <time.h>
#include <stdlib.h>

int main(int argc,char** argv)
{
	clock_t start_t,end_t;
	start_t = clock();

	int start = atoi(argv[1]);
	int end   = atoi(argv[2]);

	int i;
	long tmp;


	for(tmp = 2 * end;;tmp += end)
		for(i = start; i <= end; i++)
			if((tmp % i) != 0)
				break;
			else if( i == end)
			{
				end_t = clock();
				printf("Ans --> %ld Elapsed --> %.3lf\n",
						tmp,
						((double)(end_t - start_t))/CLOCKS_PER_SEC);
				return 0;
			}
			
}

File: euler/prob5.py
import sys,time
def prob5(start,end):
	start_t = time.time()
	tmp = 2 * end
	while 1:
		for i in range(start,end+1):
			if (tmp %i ) != 0:
				break
			elif i == end:
				print "Ans --> %d Elapsed Time --> %f" %(tmp,time.time() - start_t)
				sys.exit(0)
		tmp = tmp + end

prob5(1,20)

